packages/docs/developer-resources/integrations/listings.md

```diff
@@ -0,0 +1,107 @@
+# Listing Guide
+Welcome to the Listing Guide documentation page. If you are a digital asset exchange or ranking site, this guide will help you run a node and audit your setup. 
+
+## Support
+If you have any questions or need assistance with these instructions, please contact cLabs or ask in the #exchanges channel on [Celo’s Discord server](https://chat.celo.org/). Remember that Discord is a public channel: never disclose recovery phrases (also known as backup keys, or mnemonics), private keys, unsanitized log output, or personal information.
+
+This guide will also help you find all the necessary information about brand assets, how to integrate with Celo and what useful listing information are made available to you as well as any information about looking for support.
+
+## Celo Brand Assets for Listing
+If you are listing Celo on your exchange, you will probably need access to the Celo Platform brand assets. They can be found [here](https://celo.org/experience/brand#overview).  
+
+Please ensure your use of the Celo Platform assets provided follows the brand policy found [here](https://celo.org/brand-policy).
+
+## How Tos
+### Integrating Celo With Your Infrastructure
+There are several ways to integrate the Celo Platform with your infrastructure.
+
+A general overview of integrations that would be relevant to you listing Celo Platform are shown [here](https://docs.celo.org/developer-guide/overview/integrations/general).
+
+For more specific use-cases for exchanges, please checkout the [Custody and Exchange Integration Guide](https://docs.celo.org/developer-guide/overview/integrations/custody) as well.
+
+## Important Information
+### Celo Native Asset and Celo Dollar
+
+There are two important assets on the Celo network, the Celo native asset (CELO) and the Celo Dollar (cUSD). CELO was called Celo Gold (cGLD) when the contract was deployed, so you will often see references to Celo Gold in the codebase. To learn more about the two, please read [this](https://docs.celo.org/developer-guide/overview/celo-for-eth-devs#the-celo-native-asset-and-the-celo-dollar) section of the docs.
```

Comment:
Might want to reference this for the renaming  https://forum.celo.org/t/proposal-to-rename-celo-gold-to-celo-native-asset/528